lib/globals.d.ts

```diff
@@ -21,3 +21,49 @@ declare module '*.json' {
   const value: Record<string, any>;
   export = value;
 }
+
+// these declarations will be removed once the types.ts file for Pep440 is ready
```

Comment:
Please move these interfaces to `lib/types/renovate_pep440.d.ts` and include this file to ts config.